src/Conda.jl

```diff
@@ -192,7 +192,11 @@ end
 const PkgOrPkgs = Union{AbstractString, AbstractVector{<: AbstractString}}
 
 "Install a new package or packages."
-function add(pkg::PkgOrPkgs, env::Environment=ROOTENV; channel::AbstractString="")
+function add(pkg::PkgOrPkgs; env::Environment=ROOTENV, channel::AbstractString="")
+    add(pkg,env,channel)
+end
+
+function add(pkg::PkgOrPkgs, env::Environment, channel::AbstractString="")
```

Comment:
Why is this 3-argument version here?

Reply:
We should just have the keyword version, plus a deprecated 2-arg version, if we want to do this.